src/movement/thruster_sabertooth.cpp

```diff
@@ -117,5 +149,12 @@ int main(int argc, char **argv)
 
   ros::spin();
 
+  uint8_t serial_data[4];
```

Comment:
If this isn't used outside of the `createThrusterPacket`, some copying could be saved by either passing a pointer or creating the array within the function.